### PR TITLE
ttyd: update to 1.5.1

### PR DESCRIPTION
**Pull request by tsl0922**
Maintainer: me
Compile tested: ramips, mt7620
Run tested: none

Description:

ttyd: update to 1.5.1

### CODE REVIEW

File: utils/ttyd/Makefile [hunk 1]
```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=ttyd
-PKG_VERSION:=1.4.2
+PKG_VERSION:=1.5.1
 PKG_RELEASE:=2
```
Comment (neheb) [inline utils/ttyd/Makefile]: set this to 1

Comment (tsl0922) [reply utils/ttyd/Makefile]: Done

File: utils/ttyd/Makefile [hunk 2]
```diff
@@ -8,12 +8,12 @@
...
-PKG_RELEASE:=2
+PKG_VERSION:=1.5.1
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/tsl0922/ttyd/tar.gz/$(PKG_VERSION)?
-PKG_HASH:=ff1a66b418df6cd741868a8ea84f69cd63f15e52e3fa117641ec57d3c37a1315
+PKG_HASH:=817d33d59834f9a76af99f689339722fc1ec9f3c46c9a324665b91cb44d79ee8
 
 PKG_LICENSE:=MIT
```
Comment (neheb) [inline utils/ttyd/Makefile]: move MAINTAINER here as PKG_MAINTAINER.

**Comment by BKPepe** [comment]
@neheb  C'mon! The CircleCI is failing all the time for this pull request!
https://circleci.com/gh/openwrt/packages/tree/pull%2F9496

It was not rebased on master, so we don't see that it can be compiled. Thus it shouldn't be merged!

**Comment by neheb** [comment]
Fair enough.

**Pull request merged**